modules/imaging_uploader/jsx/log_panel.js

```diff
@@ -0,0 +1,175 @@
+/* global UploadProgress */
+
+class LogPanel extends React.Component {
```

Comment:
nit: The filename should probably be consistent with the class name. (One is `log_panel` the other is `LogPanel`)